src/pre_defined_classes/simple_programs_taking_numeric_user_input/hard/q6/MainTest.java

```diff
@@ -0,0 +1,55 @@
+package pre_defined_classes.simple_programs_taking_numeric_user_input.hard.q6;
+
+import global.BaseTest;
+import global.exceptions.InvalidClauseException;
+import global.tools.TestOption;
+import global.variables.Clause;
+import global.variables.clauses.*;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+
+public class MainTest extends BaseTest {
+    // Parsons with distractors
+    public Clause[] testSentence(){
+        TestOption.isInputTest = true;
+        TestOption.defaultInput = "12345" + System.lineSeparator() + "54321" + System.lineSeparator();
+        return new Clause[]{
+                new StringLiteral("Enter a US zip code: "),
+                new NewLine(),
+                new StringLiteral("Re-enter the US zip code: "),
+                new NewLine(),
+                new StringLiteral("The first zip code is: "),
+                new PlaceHolder(),
```

Comment:
You should just change the PlaceHolder() to IntegerLiteral(), and then have an assert statement in the test .